proc/vcm.go

```diff
@@ -51,38 +56,30 @@ func ParseConfig(filename string) (*Configuration, error) {
 	}
 
 	// Set default values for measurements if not specified
-	for i := range GswConfig.Measurements {
+	for k := range GswConfig.Measurements {
```

Comment:
Why did you switch the var?

Reply:
originally, I had it as
`for k, v := range`
as I was using the v for debugging
and generally imo the k makes more sense, as it is iterating through the keys of the map, whereas before the i made sense as it was iterating through the indexes of the list

Reply:
Fair